src/Sarif/Autogenerated/SarifRewritingVisitor.cs

```diff
@@ -756,13 +756,11 @@ public virtual Run VisitRun(Run node)
 
                 if (node.LogicalLocations != null)
                 {
-                    var keys = node.LogicalLocations.Keys.ToArray();
-                    foreach (var key in keys)
+                    if (node.LogicalLocations != null)
```

Comment:
>if (node.LogicalLocations != null) [](start = 20, length = 34)

No need to repeat the test from Line 757. I see that this is a copy of the hand-coded file in the NotYetAutogenerated directory.